os/arch/arm/src/amebad/amebad_i2s.c

```diff
@@ -0,0 +1,1818 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
Please change the year to 2021.

Reply:
yes, I'll change for all the newly added files.

Reply:
@jeongchanKim , Do we have any TC to test this porting?

Reply:
yes, you can test it by enabling i2c TCs under the sysio_itc.
filename:  `apps/examples/testcase/ta_tc/systemio/itc/itc_i2c.c`